test/e2e/parallel/user_workloads_test.go

```diff
@@ -31,15 +31,17 @@ func TestIdlerAndPriorityClass(t *testing.T) {
 	hostAwait := await.Host()
 	memberAwait := await.Member1()
 
-	// start the AAP workloads at first, because the idler timeout is longer for this case
-	aapIdler, _ := prepareIdlerUser(t, await, "test-aap-idler")
+	// create a special idler for the workloads whose pods won't be removed - their operators are not running in the cluster
```

Comment:
```suggestion
	// create a special idler for the workloads whose pods won't be removed in this e2e environment because  the corresponding operators (like AAP, RHOAI) are not running in the test cluster. The idler is expected to mark the corresponding pod's owners (aap, notebooks, etc. CRs) as to-be-idled but there is no operators to act upon it and actually delete the pods. 
```